### ISSUE TITLE
FileField should preserve alt text

### ISSUE THREAD

**Issue by stevenle** [blinkk/rootjs#647]
**Describe the bug**

When replacing an image or video, the existing alt text gets cleared. It should be preserved when the file changes.
